scudcloud-1.0/lib/wrapper.py

```diff
@@ -106,6 +106,12 @@ def _linkClicked(self, qUrl):
         else:
             self.systemOpen(url)
 
+    def gotoNextOpenChannelOrIM(self, unreads_only=False, backwards=False):
+        return self.page().currentFrame().evaluateJavaScript(
+            'TS.client.ui.gotoNextOpenChannelOrIM(%s, %s);' % (
+                str(bool(unreads_only)).lower(),
+                str(bool(backwards)).lower()))
+
```

Comment:
Hi @thepaul and thanks for the great code!

Can I just ask you something? I'm trying to avoid expose the `TS` object directly to ScudCloud: I'm wrapping all calls to it inside the `scudcloud.js` file.

So, can you edit `scudcloud-src.js` to create a JS method that will wrap `TS.client.ui.gotoNextOpenChannelOrIM`? (Then see the `CONTRIBUTING.md` to check how to minimize it).